src/sentry/options/defaults.py

```diff
@@ -26,7 +27,7 @@
 # Absolute URL to the sentry root directory. Should not include a trailing slash.
 register('system.url-prefix', ttl=60, grace=3600, flags=FLAG_REQUIRED | FLAG_PRIORITIZE_DISK)
 register('system.root-api-key', flags=FLAG_PRIORITIZE_DISK)
-register('system.logging-format', default='human', flags=FLAG_PRIORITIZE_DISK)
+register('system.logging-format', default=formats.HUMAN, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
This isn't great. We'd want it to still be the string. This would mean in your config file, you'd have to do:

```yaml
system.logging-format: 1
```

Which won't mean anything to a user.